services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HakemusyhteystietoFactory.kt

```diff
@@ -0,0 +1,49 @@
+package fi.hel.haitaton.hanke.factory
+
+import fi.hel.haitaton.hanke.allu.CustomerType
+import fi.hel.haitaton.hanke.application.ApplicationContactType
+import fi.hel.haitaton.hanke.application.ApplicationEntity
+import fi.hel.haitaton.hanke.hakemus.HakemusyhteystietoEntity
+import fi.hel.haitaton.hanke.permissions.HankekayttajaEntity
+import org.springframework.stereotype.Component
+
+@Component
+class HakemusyhteystietoFactory {
```

Comment:
This and HakemusyhteyshenkiloFactory could be objects since they only contain the companion object.

Reply:
Good call! While changing these back and forth and adding and removing save / create functions these got left as classes. I will change them as objects.